src/libs/PopUpNotification.js

```diff
@@ -0,0 +1,138 @@
+import React, {
+    forwardRef, useImperativeHandle, useRef, useState,
+} from 'react';
+import {
+    StyleSheet, Text, View, Animated, Platform,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../styles/colors';
+import Icon from '../components/Icon';
+import {Checkmark, Exclamation} from '../components/Icon/Expensicons';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../components/withWindowDimensions';
+
+const desktopContainerStyle = {
+    maxWidth: '380px',
+    top: '20px',
+    right: 0,
+    position: 'fixed',
+};
+
+const popupStyles = StyleSheet.create({
+    container: {
+        flex: 1,
+        justifyContent: 'flex-start',
+        position: 'absolute',
+        width: '100%',
+        zIndex: 2,
+        ...Platform.select({
+            web: desktopContainerStyle,
+            macos: desktopContainerStyle,
+            windows: desktopContainerStyle,
+        }),
+    },
+    smallScreenWidth: {
+        maxWidth: 'none',
+    },
+    box: {
+        backgroundColor: colors.dark,
+        borderRadius: 8,
```

Comment:
I believe we have a variable you can use here for the standard border radius.

Reply:
Fixed.